frontend/src/pages/Account.tsx

```diff
@@ -1,37 +1,51 @@
-import React, { useMemo, useState } from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import useAsyncEffect from 'use-async-effect';
 import { useParams } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
-
-import { useAccount } from 'src/hooks';
+import {
+  fetchAccount,
+  Loading,
+  useAppDispatch,
+  useAppSelector,
+} from 'src/store';
+import {
+  getAccount,
+  getAccountErrorMessage,
+  getAccountLoadingStatus,
+} from 'src/store/selectors/account-selectors';
```

Comment:
Could these be exported directly from the store?  In Amag and factoring I added aliases, one including the store index.  Planning on adding that here when I get the time